motionplan/planManager.go

```diff
@@ -144,7 +144,7 @@ func (pm *planManager) planAtomicWaypoints(
 			return nil, ctx.Err()
 		default:
 		}
-		pm.logger.Info("planning step", i, "of", len(waypoints), ":", wp.goalState)
+		pm.logger.Info("planning step ", i, "of ", len(waypoints), ":", wp.goalState)
```

Comment:
This already manifests as:
`planning step 17 of 20:`

We shouldn't need to double up spacing.